lib/hap/services/air_quality_sensor.ex

```diff
@@ -0,0 +1,32 @@
+defmodule HAP.Services.AirQualitySensor do
+  @moduledoc """
+  Struct representing an instance of the `public.hap.service.sensor.air-quality` service
+  """
+
+  defstruct air_quality: nil, name: nil, active: nil, fault: nil, tampered: nil, low_battery: nil
+
+  defimpl HAP.ServiceSource do
+    def compile(value) do
+      HAP.Service.ensure_required!(__MODULE__, "air_quality", value.air_quality)
+
+      %HAP.Service{
+        type: "8A",
+        characteristics: [
+          {HAP.Characteristics.AirQuality, value.air_quality},
+          {HAP.Characteristics.Name, value.name},
+          # TODO: The following optional characteristics still have to be implemented
+          # ”9.71 Ozone Density” (page 191)
+          # ”9.64 Nitrogen Dioxide Density” (page 189)
+          # ”9.106 Sulphur Dioxide Density” (page 221)
+          # ”9.66 PM2.5 Density” (page 190)
+          # ”9.72 PM10 Density” (page 192)
+          # ”9.126 VOC Density” (page 233)
```

Comment:
Is it possible to complete these characteristics?

Reply:
Yes, I had it already planned, just wanted to open the PR so you know that this is WIP.